src/controller/TestEventTriggerDelegate.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I don't really know the best spot for this right now. I didn't think `src/platform` was any better, but based on other files it did seem like a slightly better place potentially. The only thing I think maybe be slightly better than this location is somewhere in `src/app/`. But I don't see a precedent that is a good example to follow